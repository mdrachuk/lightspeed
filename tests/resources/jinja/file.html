<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>{{ source.filename }}</title>
</head>
<body>
filename: {{ source.filename }}
filename.name: {{ source.filename.stem }}
filename.extension: {{ source.filename.extension }}

</body>
</html>