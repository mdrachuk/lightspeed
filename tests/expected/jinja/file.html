<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>file.html</title>
</head>
<body>
filename: file.html
filename.name: file
filename.extension: html

</body>
</html>